components/brave_news/browser/feed_v2_builder.cc

```diff
@@ -948,14 +1085,24 @@ mojom::FeedV2Ptr FeedV2Builder::GenerateAllFeed() {
   const auto& publishers = publishers_controller_->GetLastPublishers();
   const auto& locale = publishers_controller_->GetLastLocale();
 
+  std::vector<std::string> subscribed_publishers;
+  for (const auto& [id, publisher] : publishers) {
+    if (publisher->user_enabled_status == mojom::UserEnabled::ENABLED) {
+      subscribed_publishers.push_back(id);
+      DVLOG(1) << "Subscribed to: " << id << std::endl;
```

Comment:
I don't think you need to push `std::endl` to `DVLOG` - it should jus tbe automatic.